lib/cli/commands/build.js

```diff
@@ -54,7 +54,7 @@ optimized output compatible with Node 6.
 
   build(dirs = this.dirs) {
     if (!dirs) {
-      dirs = this.dirs = [ 'app', 'config' ];
+      dirs = this.dirs = [ 'app', 'config', 'blueprints' ];
```

Comment:
Yea - maybe change this block to use globs instead of blindly recursing down each parent dir? Then we can change `this.dirs` => `this.globs`, and set it to: `[ 'app/**/*', 'config/**/*', 'blueprints/*/index.js' ]`